unit_1d_hw.py

```diff
@@ -7,6 +7,128 @@
 '''
 import random # We will talk about imports later :)
 
+'''
+DEV NOTES:
+
+For Unit 1D They learn te below:
+
+- Functions 
+    - Creating a function
+    - Returning a value (including void)
+    - Creating parameters for function
+    - Default arguments
+    - Multiple returns
+
+- Classes
+    - calling .append on a list (may need to use a fixture for this)
+    - Creating an object 
+    - Creating the init function
+    - Creating more functions in the same class
```

Comment:
I noticed these last three don't appear to be in the homework yet. Is that purposeful? Or will there be another homework for the unit?